pandas/io/excel.py

```diff
@@ -111,12 +111,16 @@
 
     .. versionadded:: 0.19.0
 
-parse_cols : int or list, default None
+usecols : int or list, default None
     * If None then parse all columns,
-    * If int then indicates last column to be parsed
-    * If list of ints then indicates list of column numbers to be parsed
+    * If int then indicates last column to be used
+    * If list of ints then indicates list of column numbers to be used
     * If string then indicates comma separated list of column names and
       column ranges (e.g. "A:E" or "A,C,E:F")
+parse_cols : int or list, default None
+    .. deprecated:: 0.21.0
```

Comment:
@TomAugspurger if we like this (and lgtm), let's make an issue to updated all DEPRECATED in codebase with this directive